python/lsst/ts/electrometer/controller.py

```diff
@@ -259,16 +260,17 @@ async def start_scan_dt(self, scan_duration):
         await self.send_command(f"{self.commands.set_buffer_size(50000)}")
         await self.send_command(f"{self.commands.select_device_timer()}")
         await self.send_command(f"{self.commands.next_read()}")
-        self.manual_start_time = time.time()
+        self.manual_start_time = time.monotonic()
```

Comment:
At line 251 you say "Start storing values to the buffer" and similarly for `start_scan`.

It took me awhile to realize the buffer was in the Keithly. Please add a few more words to that effect to *both* doc strings. For this one it could be as simple as "Start storing values in the Kiethley electrometer, for a set duration".

Reply:
Done by me since Eric is on vacation.